Changes.md

```diff
@@ -15,6 +15,11 @@ Features
 
 - LightTool : Added manipulator for quad lights.
 - Collect : Added a utility node for collecting the values of arbitrary inputs across a range of contexts.
+- 3Delight :
+  - Added support for VDB volume objects.
+  - Added support for `vdbVolume` shader.
+  - Added support for `volumeshader` and `displacementshader` attributes.
+  - Added support for spline parameters in shaders.
```

Comment:
This is a pain I know, but these need moving to the new `1.3.x` section now that we've released `1.3.2.`

Reply:
No worries, fixed in the edited commits throughout.